packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -144,7 +146,8 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
         ),
         Expanded(
           child: Split(
-            axis: Axis.horizontal,
+            axis: aspectRatio >= 1 ? Axis.horizontal : Axis.vertical,
```

Comment:
Lets add a widget for this functionality rather than just adding it to the inspector screen even though Flutter does make it really simple to just add the functionality like this. We'll need this shared widget elsewhere in the UI and maybe at some point we'll add a fancy animation to make the transition between split directions like nice.

Reply:
I suspect that a widget for this would just be an abstraction around the one-liner assignment to axis.

We can already have the Split animate itself when the value of `widget.axis` changes, so we can treat the animation as orthogonal to the structure of the layout.

Reply:
Added the widget, ready for another round of review